README.md

```diff
@@ -4,6 +4,8 @@
 # Overview
 Windows Container Tools is a collection of tools to augment the Windows Container experience.
 
+For general discussions and issues surrounding Windows Containers please visit https://github.com/microsoft/Windows-Containers.

```

Comment:
nit: you can remove this since it's un-related. Will also introduce merge conflict since it's already in `main`.